src/transport/udp_transport.cpp

```
@@ -96,11 +96,23 @@ MessagePtr UdpTransport::receive_message() {
         return nullptr;
     }
 
-    MessagePtr message = receive_queue_.front();
+    auto [message, sender] = receive_queue_.front();
```

Cppcheck (2.21.0)

[style] The function 'payload_view' is never used.

(unusedFunction)